src/.nuget/packageLoad.targets

```diff
@@ -38,31 +38,17 @@
   <Target Name="_RestoreBuildToolsPackagesConfig"
```

Comment:
In the glorious future, we'll just do "dotnet restore". For now, I'm just letting this file diverge between xplat and master.